src/en/temp-release-notes.md

```diff
@@ -585,15 +585,14 @@ when deploying individual charms. The bundle YAML can include a section
 called "bindings", defining the map of endpoint names to space names.
 
 Example bundle.yaml excerpt:
-    ...
+
```

Comment:
no, it's fine. they are only temporary after all - I just thought as you were fixing it, you might as well fix it nicely :)